src/server/ability.ts

```diff
@@ -1,42 +1,42 @@
 import {AbilityBuilder, Ability} from '@casl/ability'
-import {loadCurrentUser} from 'lib/users'
 import {intersection, isString} from 'lodash'
+import {loadCurrentViewerRoles} from '../lib/viewer'
 
 type Actions = 'manage' | 'upload'
 type Subjects = 'Video' | 'all'
+export type Roles = 'admin' | 'editor' | 'publisher'
```

Comment:
this gives type checking which is nice, only drawback was having to cast in the tests